### DISCUSSION TITLE
Just to say thanks

### DISCUSSION THREAD

**Discussion by 00sapo** [ivan-hc/AM#998] [General]
Just wanted to say thanks. Your work just replaced all my flatpaks. 

I've always hated the flatpak way of running apps from cli, as well as all those limits on the containerization that makes 50% of the apps not working properly.

 Moreover, some apps are not distributed as flatpaks (in my case, Beeper and and a few others). 

Not least, AM allows to use apps that I was used to install via external apt  in a cleaner way, without affecting the base OS, without compromising performances due to containers, and without relying on unofficial builds (e.g. browser, zotero, and so on)

**Comment by ivan-hc** [comment]
Grazie a te, sono contento che apprezzi il lavoro che faccio.

PS: si, sono italiano anch'io xD
